cli/get.go

```diff
@@ -76,6 +78,13 @@ A role must be specified when using this command through the --role flag. You ma
 			label = account.Name
 		}
 
+		var credentials AWSCredentials
```

Comment:
This is the main meat of this changeset which ensures that we do not re-fetch credentials if the credentials in the environment are due to expire in less than `timeRemaining` minutes